jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -594,3 +595,31 @@ def _update_stretch_histogram(self, msg={}):
         if len(sub_data) > 0:
             hist_lims = interval.get_limits(sub_data)
             hist_mark.min, hist_mark.max = hist_lims
+
+        self._check_if_v_stretch_changed()
+
+    @observe('stretch_vmin_value', 'stretch_vmax_value')
+    def _check_if_v_stretch_changed(self, msg=None):
+        self._remove_histogram_marks()
+        self._add_histogram_marks()

```

Comment:
can we just modify the position of the existing marks instead of destroying and creating new ones?